### PR TITLE
Exclude partitions from data dictionary

### PR DESCRIPTION
**Pull request by siraj-mx51**
Partitions have the same schema as header table, this PR excludes them to remove redundancy from data.json

### CODE REVIEW

File: action/store/postgres.py [hunk 1]
```diff
@@ -31,6 +31,9 @@ def read(self) -> dict:
                 c.data_type,
                 tc.constraint_type = 'PRIMARY KEY' as is_primary_key
             from information_schema.columns as c
+				join pg_class as pgc
```
Comment (gouline) [inline action/store/postgres.py]: Please align indentation for both diffs with the lines below it

**Review by gouline — COMMENTED** [review]
Looks good overall, other than the indentation bit, but I'll wait for a confirmation that it works with your target repo PR against this feature branch

File: action/store/postgres.py [hunk 2]
```diff
@@ -41,7 +44,9 @@ def read(self) -> dict:
                     and kcu.table_schema = tc.table_schema
                     and kcu.table_name = tc.table_name 
                     and kcu.constraint_name = tc.constraint_name
-            where 
+            where
+                pgc.relispartition = false and
+                pgc.relkind in ('r','v','m','p') and
                 c.table_schema not in ('information_schema', 'pg_catalog')
```
Comment (gouline) [inline action/store/postgres.py]:
```suggestion
                pgc.relispartition = false
                and pgc.relkind in ('r', 'v', 'm', 'p')
                and c.table_schema not in ('information_schema', 'pg_catalog') 
```

**Comment by siraj-mx51** [comment]
> Looks good overall, other than the indentation bit, but I'll wait for a confirmation that it works with your target repo PR against this feature branch

@gouline re-tested in run https://github.com/mx51/msp-transaction-service/actions/runs/4359110337/jobs/7620518992

**Review by gouline — APPROVED** [review]

**Pull request merged**